schema.sql

```diff
@@ -0,0 +1,156 @@
+DROP DATABASE IF EXISTS career_lattice;
+CREATE DATABASE career_lattice;
+USE career_lattice;
+
+-- ---------------------------------- TABLES ------------------------- --
+
+CREATE TABLE user (
+    user_id INT PRIMARY KEY AUTO_INCREMENT,
+    username VARCHAR(255) NOT NULL,
+    password VARCHAR(255) NOT NULL,
+    salt VARCHAR(255) NOT NULL,
```

Comment:
Is there a reason why there's a separate column for the salt? I would have thought the `password` column contains the plaintext password + salt, which is then all hashed together?